{
	"schema_version": "1.6.0",
	"id": "CVE-2020-25668",
	"modified": "2024-11-21T05:18:24Z",
	"published": "2021-05-26T12:15:15Z",
	"aliases": [
		"A-190228658",
		"PUB-A-190228658"
	],
	"related": [
		"DLA-2483-1",
		"DLA-2494-1",
		"SUSE-SU-2020:3484-1",
		"SUSE-SU-2020:3503-1",
		"SUSE-SU-2020:3507-1",
		"SUSE-SU-2020:3512-1",
		"SUSE-SU-2020:3513-1",
		"SUSE-SU-2020:3522-1",
		"SUSE-SU-2020:3532-1",
		"SUSE-SU-2020:3544-1",
		"SUSE-SU-2020:3648-1",
		"SUSE-SU-2020:3651-1",
		"SUSE-SU-2020:3670-1",
		"SUSE-SU-2020:3690-1",
		"SUSE-SU-2020:3698-1",
		"SUSE-SU-2020:3713-1",
		"SUSE-SU-2020:3715-1",
		"SUSE-SU-2020:3717-1",
		"SUSE-SU-2020:3748-1",
		"SUSE-SU-2021:0437-1",
		"SUSE-SU-2021:14630-1",
		"UBUNTU-CVE-2020-25668",
		"USN-4679-1",
		"USN-4680-1",
		"USN-4681-1",
		"USN-4751-1",
		"openSUSE-SU-2020:1906-1",
		"openSUSE-SU-2020:2112-1",
		"openSUSE-SU-2020:2260-1",
		"openSUSE-SU-2021:0242-1",
		"openSUSE-SU-2024:10728-1",
		"openSUSE-SU-2024:13704-1"
	],
	"details": "A flaw was found in Linux Kernel because access to the global variable fg_console is not properly synchronized leading to a use after free in con_font_op.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:L/AC:H/PR:L/UI:N/S:U/C:H/I:H/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:11",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.9.6-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2020-25668.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:12",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.9.6-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2020-25668.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:13",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.9.6-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2020-25668.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://security.netapp.com/advisory/ntap-20210702-0005/"
		},
		{
			"type": "WEB",
			"url": "http://www.openwall.com/lists/oss-security/2020/10/30/1"
		},
		{
			"type": "WEB",
			"url": "http://www.openwall.com/lists/oss-security/2020/11/04/3"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit?id=90bfdeef83f1d6c696039b6a917190dcbbad3220"
		},
		{
			"type": "WEB",
			"url": "https://lists.debian.org/debian-lts-announce/2020/12/msg00015.html"
		},
		{
			"type": "WEB",
			"url": "https://lists.debian.org/debian-lts-announce/2020/12/msg00027.html"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1893287%2C"
		},
		{
			"type": "WEB",
			"url": "https://www.openwall.com/lists/oss-security/2020/10/30/1%2C"
		},
		{
			"type": "WEB",
			"url": "https://www.openwall.com/lists/oss-security/2020/11/04/3%2C"
		},
		{
			"type": "ADVISORY",
			"url": "https://security-tracker.debian.org/tracker/CVE-2020-25668"
		}
	]
}
